examples/benchmarks/odspsnapshotfetch-perftestapp/tsconfig.json

```
@@ -1,7 +1,9 @@
 {
-	"extends": "@fluidframework/build-common/ts-common-config.json",
+	"extends": [
+		"../../../common/build/build-common/tsconfig.base.json",
+		"../../../common/build/build-common/tsconfig.cjs.json",
```

```suggestion
		"../../../common/build/build-common/tsconfig.esm-only.json",
```

This package appears to only build ESM, so use the esm-only config (even though it's the same as the other I think). Also update the package.json to set type: module to further clarify it's ESM-only.

---

Makes-sense.  Thx!